### DISCUSSION TITLE
A new sound source like voice chat for the player.

### DISCUSSION THREAD

**Discussion by MidnightTokyo** [SmartlyDressedGames/Unturned-3.x-Community#3838] [Ideas]
You don't care about the forum, so I'll duplicate it here.

Add a new source of sound to the player and the ability to send him sound packets as in voice chat, to play a custom sound that only he will hear with the ability to change the volume from the server. For example, based on the volume change, you can realize a radio and depending on the player’s distance, lower the volume or stop sending packets altogether.

**Comment by Gamingtoday093** [comment]
You can already sorta do this with Effects, although with much less control over the actual audio being played. (It has to be recorded and bundled before hand)

**Reply by MidnightTokyo** [reply]
Do you know what desync is?
If a new player joins, he will not have an effect, it must be created, but the effect cannot be set any parameters from the server, because of which he will have music playing from the beginning, while the music of others has finished.

**Reply by MidnightTokyo** [reply]
Also, the main goal is to be able to play any sounds, not pre-recorded in the modpack, if you do not get it
